looker_deployer/commands/deploy_boards.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 import logging
-from looker_sdk import models
+from looker_sdk import models40 as models
```

Comment:
This will be good until the python SDK is updated after 3.1 is removed. Then models40 will probably be called `models`. We should discuss/research how much code this would break ... or maybe we alias models for models40. Worth thinking about @christelilaka @drstrangelooker et al